modules/ROOT/pages/getting-started/index.adoc

```diff
@@ -4,17 +4,14 @@
 
 
 This tutorial walks you through creating a new project with the Neo4j GraphQL Library.
-
-If you are not familiar with Neo4j and GraphQL, you can alternatively take the course https://graphacademy.neo4j.com/courses/graphql-basics/?ref=docs[Introduction to Neo4j & GraphQL] in GraphAcademy to learn the fundamentals, how to use the xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox] and the Neo4j GraphQL Library to create GraphQL APIs backed by a Neo4j graph database.
```

Comment:
It was requested before that we added the link to this GraphAcademy course in documentation, so I think it's better to leave it here

Reply:
restored